pkg/bpf/bpf_kmesh_l4_workload.go

```diff
@@ -59,15 +66,29 @@ func (l *BpfLoader) StartWorkloadMode() error {
 		return err
 	}
 
+	defer func() {
+		if err != nil {
+			l.Stop()
+		}
+	}()
+
 	if err = l.workloadObj.Load(); err != nil {
-		l.Stop()
 		return fmt.Errorf("bpf Load failed, %s", err)
 	}
 
 	if err = l.workloadObj.Attach(); err != nil {
-		l.Stop()
 		return fmt.Errorf("bpf Attach failed, %s", err)
 	}
+
+	if err = l.workloadObj.ApiEnvCfg(); err != nil {
+		return fmt.Errorf("failed to set api env")
+	}
+
+	ret := C.deserial_init()
```

Comment:
nit: can you move to bpf.go, for ads it is done there

Reply:
you mean move StartWorkloadMode to bpf.go？

Reply:
I mean the deserial_init to `StartWorkloadMode `, like ads